configure.ac

```diff
@@ -3165,12 +3165,45 @@ if test "$have_glibc_memmove_bug" = yes; then
     AS_VAR_APPEND([LIBMPDEC_CFLAGS], [" -U_FORTIFY_SOURCE"])
 fi
 
+dnl Check for SQLite library. Use pkg-config if available.
+PKG_CHECK_MODULES(
+  [LIBSQLITE3], [sqlite3 >= 3.7.15], [], [
+    LIBSQLITE3_LIBS="-lsqlite3"
+    LIBSQLITE3_CFLAGS=
+  ]
+)
+AS_VAR_APPEND([LIBSQLITE3_CFLAGS], [' -I$(srcdir)/Modules/_sqlite'])
```

Comment:
Do single quotes perform quoting of ``$`` symbol? I used ``\$(srcdir)`` in other places. Single quotes are nicer, though.

Reply:
Yeah, I prefer single quotes to escape escaping :)

Reply:
Yeah, me too. I was not aware that single quotes work here. I'll update my other code later.

Reply:
https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html  Section 2.2 Quoting